src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -25,6 +25,8 @@ import SelectJobCard from './SelectJobCard';
 import TagCloud from '../TagCloud';
 
 import { fixedEncodeURIComponent } from '../../utils/common';
+import { useDefaultSearchFilters } from '../search/data/hooks';
+import { UserSubsidyContext } from '../enterprise-user-subsidy';
```

Comment:
Since we don't to pass filters in configure for skills dropdown so there will be no need of it as well.

Reply:
Please refer to my response above.